modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase

```diff
@@ -262,12 +329,21 @@ definition {
 	}
 
 	@description = "COMMERCE-5862. As a buyer, I want to view the product SKU, if set(single SKU)"
-	@ignore = "true"
 	@priority = "3"
 	test ViewProductCardSKU {
 		property portal.acceptance = "false";
+		property test.name.skip.portal.instance = "CommerceProductCard#ViewProductCardSKU";
+
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		Click(
+			key_productName = "U-Joint",
+			locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
 
-		// TODO COMMERCE-5862 ViewProductCardSKU pending implementation
+		AssertTextEquals(
+			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU",
+			value1 = "MIN55861");
```

Comment:
@pedro-michel Also here, the target is the product card, not the detail.

Reply:
Done